raiden-ts/src/messages/utils.ts

```diff
@@ -322,9 +323,10 @@ export function decodeJsonMessage(text: string): Signed<Message> {
 export async function signMessage<M extends Message>(
   signer: Signer,
   message: M,
+  { log }: { log: logging.Logger } = { log: logging },
```

Comment:
```suggestion
  { log }: { log: Logger } = { log: logging },
```

Reply:
If you make the change you suggested, there's no such `logging` var

Reply:
Whoops